release-content/migration-guides/bevy_render_reorganization.md

```diff
@@ -3,7 +3,7 @@ title: `bevy_render` reorganization
 pull_requests: [19997, 19991, 20000, 19949, 19943, 19953]
 ---
 
-You must now import `ToNormalizedRenderTarget` to use `RenderTarget::normalize`
+You must now import `bevy_render::NormalizedRenderTargetExt` to use methods on NormalizedRenderTarget
```

Comment:
reviewer note: NormalizedRenderTarget is covered by a glob re-export of camera in bevy_render::camera, so this type moving crate is transparent.